src/transxchange/read.rs

```diff
@@ -168,15 +186,105 @@ fn load_companies(transxchange: &Element) -> Result<CollectionWithId<Company>> {
     Ok(companies)
 }
 
+fn load_commercial_physical_modes(
+    transxchange: &Element,
+) -> Result<(CommercialMode, PhysicalMode)> {
+    let raw_mode = transxchange
+        .try_only_child("Services")?
+        .try_only_child("Service")?
+        .try_only_child("Mode")?
+        .text();
+    let mode = MODES
+        .get(raw_mode.as_str())
+        .ok_or_else(|| {
+            format_err!(
```

Comment:
As seen together apply the default bus mode

Reply:
Done in 9837c714c2044c279627152b80c554924f5198d8.